tests/subgraph_program_builder.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2023 CINN Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "tests/program_builder.h"
+
+namespace cinn {
+namespace tests {
+
+class BiasBnReLUSubGraphBuilder : public ProgramBuilder {
```

Comment:
```suggestion
class BiasBnReLUBuilder : public ProgramBuilder {
```
`xxSubGraphBuilder`-> `xxBuilder`，以下类似

Reply:
done